rules/Renaming/Rector/Name/RenameClassRector.php

```diff
@@ -30,9 +31,10 @@ final class RenameClassRector extends AbstractRector implements ConfigurableRect
     public const OLD_TO_NEW_CLASSES = 'old_to_new_classes';
 
     /**
-     * @var array<string, string>
+     * @api
+     * @var string
      */
-    private array $oldToNewClasses = [];
+    public const CLASS_MAP_FILES = 'class_map_files';
```

Comment:
@sabbelasichon I took the liberty of adding your map alias feature here :)  I think it will be very useful to have option to outsource renamed classes to external file, as they can be re-used elsewhere.

Reply:
You are allowed to copy whatever and whenever you like.